### PR TITLE
Add KCSA Certification to Training page

### PR DESCRIPTION
**Pull request by Community-Programmer**
Added KCSA Certification to Training page

- Added KCSA Logo at Section page
- Added KCSA Certification information

img
img2

### CODE REVIEW

**Review by aj11anuj — COMMENTED** [review]
LGTM, 👍

File: content/en/training/_index.html
Comment (Gauravpadam) [inline content/en/training/_index.html:19]: I'd move this div towards the end of logo divs

**Comment by windsonsea** [comment]
The [preview](https://deploy-preview-43318--kubernetes-io-main-staging.netlify.app/training/) looks good, but I'm unsure which sequence would be better. Should we place two "security" icons adjacent to each other?

**Comment by Community-Programmer** [comment]
yes I think they should be adjacent to each other.

This is how its look at end
image

Also removed line breaks in it, now they are not aligned
image

**Comment by aj11anuj** [comment]
> The [preview](https://deploy-preview-43318--kubernetes-io-main-staging.netlify.app/training/) looks good, but I'm unsure which sequence would be better.
> 
> * Would it be better to have two "security" icons placed next to each other?
> * Alternatively, placing this new icon in the 2nd position might be more suitable.

+1 to 2nd point, as this certification is a beginner level certification.. it's more suitable at 2nd place (just after KCNA). For icons as well as tiles order from left to right will be more suitable as (KCNA, KCSA, CKAD, CKA and then CKS).

**Comment by Gauravpadam** [comment]
- I'm good with either position for the icon. Since both reviewers prefer 2nd position, Let's go with that
- The spacing can be adjusted with some css changes, That can come as a different PR

**Comment by sftim** [comment]
Please don't mention issue references in PR titles

/retitle Add KCSA Certification to Training page

**Comment by sftim** [comment]
We should try to make the page render better for narrow viewports - see this screenshot for why
screenshot

@Community-Programmer do you want to add a commit to do that? If not, we can look to merge this and then file a new issue about making the page responsive.

**Comment by Community-Programmer** [comment]
> We should try to make the page render better for narrow viewports - see this screenshot for why screenshot
> 
> @Community-Programmer do you want to add a commit to do that? If not, we can look to merge this and then file a new issue about making the page responsive.

Yes, I will fix that too. I will add a commit for the previous one and this one together.

File: content/en/training/_index.html
Comment (sftim) [inline content/en/training/_index.html:131]: nit: maybe put this _before_ the CKS?

**Comment by Community-Programmer** [comment]
**Changes Made:**

- Changed the Position of logo of KCSA to 2nd Position
- Added CSS   to make the page render better for narrow viewports
- Removed Extra Spaces and <Br> tags

Images attached:

image
image
image

**Comment by sftim** [comment]
This looks OK to merge. Please squash it to 2 commits:
- the CSS change
- adding the KCSA

---

Even better (**optionally**):

change from:
before

to more like:
after

File: static/css/training.css
Comment (Gauravpadam) [inline static/css/training.css:174]: I would target the button here, specifically `a.button` to prevent overriding

File: static/css/training.css
Comment (Gauravpadam) [inline static/css/training.css:184]: same feedback is applicable here

**Review by Gauravpadam — COMMENTED** [review]
Thanks

Here's some more feedback

**Comment by Community-Programmer** [comment]
**Changes Made:**

- Changed the logo placement of cta.
- Added responsiveness.

image

**Review by aj11anuj — APPROVED** [review]
New icon alignment with responsiveness of icons and tiles, LGTM 👍

Squash all your commits into 2 commits:-
- All "CSS change" related commits into 1
- All "Addition of KCSA" related commits into 1

**Comment by sftim** [comment]
/hold

Unusually, we'd like this squashed to 2 commits. The existing commit message “_Solved kubernetes#43317 KCSA Certification to Training page_” should **not** appear in the final commit list.

OK to unhold once that's sorted.

If you want @Community-Programmer, a maintainer can do that squash for you and update your fork accordingly. Let us know if you'd like that.

**Comment by Community-Programmer** [comment]
> /hold
> 
> Unusually, we'd like this squashed to 2 commits. The existing commit message “_Solved #43317 KCSA Certification to Training page_” should **not** appear in the final commit list.
> 
> OK to unhold once that's sorted.
> 
> If you want @Community-Programmer, a maintainer can do that squash for you and update your fork accordingly. Let us know if you'd like that.

@sftim Yes, I would greatly appreciate it, please add a maintainer to do the squash and update my fork accordingly.Thank you

**Comment by sftim** [comment]
Thanks, I squashed the commits and force pushed.

/lgtm
/hold cancel

**Comment by Community-Programmer** [comment]
> [APPROVALNOTIFIER] This PR is **NOT APPROVED**
> 
> This pull-request has been approved by: _[aj11anuj](https://github.com/kubernetes/website/pull/43318#pullrequestreview-1660461883)_ **Once this PR has been reviewed and has the lgtm label**, please assign [reylejano](https://github.com/reylejano) for approval. For more information see [the Kubernetes Code Review Process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process).
> 
> The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=kubernetes%2Fwebsite).
> 
> Needs approval from an approver in each of these files:
> * **[OWNERS](https://github.com/kubernetes/website/blob/main/OWNERS)**
> 
> Approvers can indicate their approval by writing `/approve` in a comment Approvers can cancel approval by writing `/approve cancel` in a comment

/assign @reylejano

**Comment by Community-Programmer** [comment]
@sftim can we merge it now??

**Comment by Gauravpadam** [comment]
@Community-Programmer I would stay patient and wait for the approvers to get to this PR, Please refrain from tagging and/or assigning them. Thanks again!

**Comment by aj11anuj** [comment]
It's almost ready to merge just a perspective of some other approver is awaiting.

**Comment by sftim** [comment]
>  I would stay patient and wait for the approvers to get to this PR, Please refrain from tagging and/or assigning them. 

The assignment in https://github.com/kubernetes/website/pull/43318#issuecomment-1751298948 was **valid**; this was following advice from @k8s-ci-robot. It's always OK to follow the robot's advice (even if the advice is actually wrong, that means there was a bug in the robot and not a mistake by the person who relied on its advice).

**Pull request merged**
